apps/e2e-tests/src/process-compose/e2e.spec.ts

```diff
@@ -39,6 +53,31 @@ describe.sequential('E2E Tests with process-compose', async () => {
     expect(equal).toBe(true);
   });
 
+  test('Test sequencer config is available and in correct format', async () => {
+    sequencerConfig = await fetchAndDecodeJSON(
+      SequencerConfigV2Schema,
+      sequencerConfigUrl,
+    );
+
+    expect(sequencerConfig).toBeTypeOf('object');
+
+    // Collect initial information for the feeds and their prices
+    const url = sequencerConfig.providers[network].url;
+    const contractAddress = sequencerConfig.providers[network]
+      .contract_address as `0x${string}`;
+    allowFeeds =
+      sequencerConfig?.providers?.[network]?.allow_feeds?.map(feedId =>
+        BigInt(feedId),
+      ) ?? [];
+
+    ADFSConsumer = AggregatedDataFeedStoreConsumer.createConsumerByRpcUrl(
+      contractAddress,
+      url,
+    );
+
+    initialPrices = await getPricesInfo(allowFeeds, ADFSConsumer);
```

Comment:
Now then we have `mapValuePromises` function we can remove the `getPricesInfo` function and get prices like this:
```suggestion
    initialPrices = allowFeeds.reduce(
      (acc, feedId) => {
        acc[feedId.toString()] = 0;
        return acc;
      },
      {} as Record<string, number>,
    );

    initialPrices = await mapValuePromises(
      initialPrices,
      async (feedId, _) =>
        await ADFSConsumer.getLatestSingleData(feedId).then(res =>
          Number(res.slice(0, 50)),
        ),
    );
```

Reply:
1. We set default value of `0` to `initialPrices` where keys are the feeds of interest
2. We set the price values, as they are in this state of the test